src/java.base/share/classes/jdk/internal/logger/BootstrapLogger.java

```diff
@@ -964,6 +983,18 @@ static Logger getLogger(LazyLoggerAccessor accessor) {
         }
     }
 
+    private static void ensureClassInitialized(Class<?> c) {
+        try {
+            MethodHandles.lookup().ensureInitialized(c);
```

Comment:
Hello Sean, should we check if there are any implications, like on startup performance, of using `MethodHandles` in this `BootstrapLogger`?

Reply:
Thanks for the comments Jai. Latest patch just pushed resolved this -  in any case, it looks like the MethodHandles class is loaded very early in the module system  (even before the application code)

```
[0.026s][info][class,load] java.util.HexFormat source: shared objects file
[0.026s][info][class,load] java.util.concurrent.atomic.AtomicInteger source: shared objects file
[0.026s][info][class,load] jdk.internal.module.ModuleBootstrap source: shared objects file
[0.026s][info][class,load] java.lang.invoke.MethodHandles source: shared objects file
[0.026s][info][class,load] java.lang.invoke.MemberName$Factory source: shared objects file
[0.026s][info][class,load] java.lang.invoke.MethodHandles$Lookup source: shared objects file
[0.027s][info][class,load] java.lang.StrictMath source: shared objects file


[0.032s][info][class,load] java.security.SecureClassLoader$DebugHolder source: shared objects file
[0.032s][info][class,load] sun.security.util.Debug source: shared objects file
[0.033s][info][class,load] SignedLoggerFinderTest source: file:/MYREPO/jdk/open/test/jdk/JTwork/classes/0/java/lang/System/LoggerFinder/SignedLoggerFinderTest/SignedLoggerFinderTest.d/
[0.033s][info][class,load] java.lang.NamedPackage source: shared objects file
[0.033s][info][class,load] jdk.internal.misc.MainMethodFinder source: shared objects file
[0.033s][info][class,load] jdk.internal.misc.PreviewFeatures source: shared objects file
```